Sources/MapboxCoreNavigation/CoreFeedbackEvent.swift

```diff
@@ -2,10 +2,10 @@ import Foundation
 import MapboxDirections
 import Polyline
 
-class CoreFeedbackEvent: Hashable {
-    let id = UUID()
-    let timestamp: Date
-
+class CoreFeedbackEvent: Hashable, Codable {
```

Comment:
Curious, why it is a class instead of struct?

Reply:
I guess this is because of inheritance, other events inherit from this class.